python/ray/train/tests/test_session.py

```diff
@@ -170,6 +171,40 @@ def test_report_after_finish(session):
     shutdown_session()
 
 
+@pytest.mark.parametrize(
+    "block,put_result_queue,put_actor_queue",
+    [
+        # Users should never report to both queues simultaneously.
+        (False, False, False),
+        (False, False, True),
+        (False, True, False),
+        (True, False, False),
+        (True, False, True),
+        (True, True, False),
+    ],
+)
+def test_get_result_from_queues(session, block, put_result_queue, put_actor_queue):
+    result_queue_training_result = _TrainingResult(
```

Comment:
short docstring describing the input bools and expectation

Reply:
done.